consensus/hotstuff/vote_aggregator.go

```diff
@@ -18,6 +18,10 @@ type BlockSigner interface {
 	// return (vote, nil) if vote is created
 	// return (nil , module.InvalidBlockError) if the block is invalid.
 	CreateVote(*model.Block) (*model.Vote, error)
+
+	// CreateProposal returns a proposal for the given block.
+	// return (proposal, nil) if vote is created
+	CreateProposal(*model.Block) (*model.Proposal, error)
```

Comment:
With this additional methods, the BlockSigner interface is now very similar to the [consensus/hotstuff/Signer interface. ](https://github.com/onflow/flow-go/blob/6550e8295209dfa2760959c3e5c0745a1c48322d/consensus/hotstuff/signer.go#L15-L24). 

I think eventually we will keep one of them